Maple2.Server.Core/proto/sync.proto

```diff
@@ -23,6 +23,10 @@ message TrophyInfo {
   int32 lifestyle = 3;
 }
 
+message ClubsInfo {
```

Comment:
naming: `ClubInfo`

Reply:
perhaps this needs a different name. ClubInfo is already used to provide a larger amount of data from World to Game. any suggestion?

Reply:
will we ever need more fields here? maybe just collapse it completely and remove this type

Reply:
It's possible. I left it open incase we need to get club names as well. I'm currently not grabbing it seeing as what I'm doing isn't totally needed at the moment.

Reply:
nvm that's also kinda inconsistent given that we use `GuildInfo` and `PartyInfo` to represent the whole Guild/Party

Reply:
What about changing the suffixes on all the sync types to `*Update` (`ClubUpdate`)